celestia/src/native.rs

```diff
@@ -3,59 +3,35 @@ use std::path::PathBuf;
 use std::time::Duration;
 
 use anyhow::{bail, Context, Result};
+use celestia_node::network::{canonical_network_bootnodes, network_genesis, network_id, Network};
 use celestia_node::node::{Node, NodeConfig};
 use celestia_node::store::SledStore;
 use celestia_rpc::prelude::*;
-use celestia_types::hash::Hash;
-use clap::{Parser, ValueEnum};
 use libp2p::{identity, multiaddr::Protocol, Multiaddr};
 use tokio::time::sleep;
 use tracing::info;
 
-#[derive(Debug, Parser)]
-struct Args {
-    /// Network to connect.
-    #[arg(short, long, value_enum, default_value_t)]
+pub(crate) async fn run(
```

Comment:
with subcommands this could be a method on `NodeArgs` and avoid passing arguments by hand too